### ISSUE TITLE
Documentation slight outdated (for quick setup)

### ISSUE THREAD

**Issue by perrochon** [moagrius/TileView#494]
Tried to get this running. Looks super nice and promising. A few things I noticed while setting it up.

1. I needed to set min sdk to 26 (Oreo) to compile. Not a problem, and easily fixable through error messages

2. Could it be that .defineZoomLevel replaced .defineDetail and .addDetailLevel? readme.md refers to defineDetail in the quick setup , and the wiki uses addDetailLevel. The demo code shows how.

**Comment by moagrius** [comment]
yeah the javadocs are completely outdated and i haven't gone through all the wiki pages.  the demo and readme should be relatively correct, although feel free to fork and PR (or just edit the wiki directly) if you have the bandwidth to correct any typos or misnamed methods.  otherwise i'll get to it when i can - i usually work in short bursts every few months.

yeah it's `defineZoomLevel(int zoom, Object data)` now.  https://github.com/moagrius/TileView/blob/master/tileview/src/main/java/com/qozix/tileview/TileView.java

**Comment by moagrius** [comment]
you should be able to edit the wiki directly, but it's np i can do it - what page/line?

**Comment by Pavle37** [comment]
I also get an error message for min sdk to be 26. How can i fix this to use it with min sdk 19?

**Comment by moagrius** [comment]
This should be in a different issue thread, but nity demo and library have minSdk of 19.  Check other Gradle or manifest settings in your project, or other dependencies

**Comment by tva-TIS** [comment]
Also in README.md `TileView.Builder().build()` does not return the build TileView anymore but void and takes the TileView to build as a paramter.

**Comment by moagrius** [comment]
i will fix this as soon as i get some time for this project (weeks away)
